app/services/idv/proofing_components.rb

```diff
@@ -2,25 +2,12 @@
 
 module Idv
   class ProofingComponents
-    def initialize(idv_session:, session:, user:, user_session:)
+    def initialize(idv_session:)
       @idv_session = idv_session
-      @session = session
-      @user = user
-      @user_session = user_session
     end
 
     def document_check
-      if user.establishing_in_person_enrollment || user.pending_in_person_enrollment
-        Idp::Constants::Vendors::USPS
-      elsif idv_session.remote_document_capture_complete?
-        DocAuthRouter.doc_auth_vendor(
-          request: nil,
-          service_provider: idv_session.service_provider,
-          session:,
-          user_session:,
-          user:,
-        )
-      end
+      idv_session.doc_auth_vendor
```

Comment:
🎉

Reply:
I'm not sure if I'm seeing it, but let's make sure I'm not just overlooking something. `doc_auth_vendor` is already being set in prod, right?

The constructor change should only be problematic if there's background job usage where what's available changes, and I didn't see any of that. I think all the controller-level action will be consistent since we're already storing this in the session in prod.

But my comment here is just intended to outline my thinking. What else should I be looking at?

Reply:
You're totally right. I was thinking in terms of jobs not the controller context. This is fine